planning/behavior_velocity_planner/include/scene_module/stop_line/scene.hpp

```diff
@@ -70,7 +74,7 @@ class StopLineModule : public SceneModuleInterface
 
 public:
   StopLineModule(
-    const int64_t module_id, const lanelet::ConstLineString3d & stop_line,
+    const int64_t module_id, const StopLineWithLaneId & stop_lines_with_lane_id,
```

Comment:
```
const int64_t module_id, const lanelet::ConstLineString3d & stop_line, const int64_t lane_id
```
I don't think std::pair is necessary since the scene module is only dealing with one pair.
Also, if we use std::pair, we have to use first and second, which reduces readability.

Reply:
[873b5e6](https://github.com/autowarefoundation/autoware.universe/pull/405/commits/873b5e610a13c50eaff383308d8de9de3dd8b6cc)